schemas/2017-06-01/storageAccounts.md

```diff
@@ -0,0 +1,187 @@
+# Microsoft.Storage/storageAccounts template reference
```

Comment:
>Microsoft.Storage/storageAccounts template reference [](start = 2, length = 52)

we don't support .md files in this repo.

Reply:
removed